### DISCUSSION TITLE
Links in the Preview Window?

### DISCUSSION THREAD

**Discussion by amreus** [junegunn/fzf#3496] [Ideas]
Is there any way to interact with any links that happen to be showing in the preview window? Either with the mouse or otherwise? Thanks.

**Comment by LangLangBart** [comment]
- try the `--no-mouse` flag
  - ref: [preview window text selection · Issue #3409 · junegunn/fzf ](https://github.com/junegunn/fzf/issues/3409#issuecomment-1692740468)

**Reply by amreus** [reply]
Oh, that work. Thank you.
